service.py

```
@@ -1,116 +1,420 @@
 # -*- coding: utf-8 -*-
 import requests
 import os
+import logging
+import json
+from datetime import datetime, timedelta
+from typing import Optional, Dict, Any
```

Ruff (0.11.9)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)